integrations/zendesk/src/channels.ts

```diff
@@ -0,0 +1,26 @@
+import { getZendeskClient } from './client'
+import { INTEGRATION_NAME } from './const'
+import { IntegrationProps } from '.botpress'
+
+export default {
+  ticket: {
+    messages: {
+      text: async ({ ...props }) => {
+        const { user } = await props.client.getUser({ id: props.payload.userId })
```

Comment:
**chore:** you can use a conversation level state that you set in `createTicket` action to store patient information